platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ConsensusImpl.java

```diff
@@ -1087,6 +1088,17 @@ private AddressBook getAddressBook() {
         return addressBook;
     }
 
+    /**
+     * Checks if the event has the same index and the one provided.
+     *
+     * @param e the event to check for euqality to index
+     * @param index the index to check for equality to the event's index
+     * @return true if the event has the same index as the one provided, false otherwise
+     */
+    private boolean hasIndex(@NonNull final EventImpl e, final int index) {
```

Comment:
I think this method could have a better name, `hasIndex` isn't very informative IMO

What do you think of `wasEventCreatedByNodeIndex`? Or maybe you can come up with a better alternative

Reply:
changed to `eventCreatorIndexMatches`